kataloger/dialogporten/opprett dialog med sykmelding.bru

```diff
@@ -10,22 +10,17 @@ post {
   auth: none
 }
 
-headers {
-  accept: application/json
-  Content-Type: application/json
-}
-
 body:json {
   {
-    "serviceResource": "urn:altinn:resource:nav_sykepenger_inntektsmelding-nedlasting",
-    "party": "urn:altinn:organization:identifier-no:315587336",
-    "externalReference": "{{externalDialogId}}",
-    "status": "New",
+    "serviceResource": "urn:altinn:resource:nav_sykepenger_dialogporten",
+    "party": "urn:altinn:organization:identifier-no:{{tigersyskundeOrgnrUnderenhet}}",
+    "externalReference": "{{eksempelSykmeldingId}}",
+    "isApiOnly": {{kunForApi}},
     "content": {
       "title": {
         "value": [
           {
-            "value": "Sykepenger for AKROBATISK PUS (f. 054494)",
+            "value": "Sykepenger for {{sykmeldtNavn}} (f. 054494)",
```

Comment:
Skulle vi inkludert f.dato i variabelen også, altså ala "sykepenger for {{ sykmeldt }}" ? Så fdato ikke er hardkodet her bare..